int32 mnId
int32 num_obs

# best refer keyframe
int64 ref_keyframe_id

# co-visible
cloud_edge_slam/Observation[] observations

# Position
geometry_msgs/Point point